### DISCUSSION TITLE
EM7455 auroflash firmware download fail

### DISCUSSION THREAD

**Discussion by James-h-Thomas** [danielewood/sierra-wireless-modems#112] [Q&A] [answered]
Afternoon all,

Sadly I can't get the autoflash.sh script to complete successfully. The script fails on trying to download the new firmware and I get:

Download of zip failed.
File size on server is too small, something is wrong, exiting...
Attempted download URL was: https://source.sierrawireless.com/~/media/support_downloads/airprime/74xx/fw/7455/zip
curl info:
HTTP/1.1 302 Found
Cache-Control: private
Content-Length: 148
Content-Type: text/html; charset=utf-8
Location: /sitecore/service/notfound.aspx
Server: Microsoft-IIS/10.0
Access-Control-Allow-Origin: *
Access-Control-Allow-Headers: Origin, X-Requested-With, Content-Type, Accept
Date: Mon, 29 Nov 2021 13:24:33 GMT
Set-Cookie: BIGipServer~SWI_CORP_WWW~SWI_CORP_CAKEL-HTTPS=928711690.47873.0000; path=/; Httponly; Secure

Any aid gratefully received!

Regards,

TW

**Comment by danielewood** [comment, marked as answer]
Sierra changed the url scheme, fixed in latest release

**Reply by rhoover9192** [reply]
Daniel,
       I am getting the following error:
Max FW images: 4
Active FW image is at slot 255
TYPE SLOT STATUS LRU FAILURES UNIQUE_ID   BUILD_ID
Max PRI images: 50
OK
AT
OK
AT!IMAGE=0
OK
AT!IMAGE?
TYPE SLOT STATUS LRU FAILURES UNIQUE_ID   BUILD_ID
FW   1    EMPTY  0   0 0
FW   2    EMPTY  0   0 0
FW   3    EMPTY  0   0 0
FW   4    EMPTY  0   0 0
Max FW images: 4
Active FW image is at slot 255
TYPE SLOT STATUS LRU FAILURES UNIQUE_ID   BUILD_ID
Max PRI images: 50
OK
Waiting for modem to reboot...
---
Download of zip failed.
File size on server is too small, something is wrong, exiting...
Attempted download URL was: https://source.sierrawireless.comzip/
curl info:
---

It looks like the URL changed again.
Thanks
Rob

**Reply by TheMHD1** [reply]
same issue happened again

**Reply by danielewood** [reply]
Fixed again in #137
